docs/guides/faq.md

```diff
@@ -80,10 +84,42 @@ yarn link @tauri-apps/api
 
 Or you can change your package.json to point to the dist folder directly:
 
-```json
+```json title=package.json
 {
   "dependencies": {
     "@tauri-apps/api": "/path/to/tauri/tooling/api/dist"
   }
 }
 ```
+
+</details>
+
+## Should I use Node or Cargo? {#node-or-cargo}
+
+Even though installing the CLI through Cargo is the preferred option, it has to compile the whole binary from scratch when you install it. If you're in a CI environment or on a very slow machine you're better off choosing another installation method.
+
+As the CLI is written in Rust, it is naturally available through [crates.io] and installable with Cargo.
+
+We also compile the CLI as a native Node.js addon and distribute it [via NPM]. This has a number of advantages compared to the Cargo installation method:
+
+1. The CLI is pre-compiled, leading to much faster install times
+2. You can pin a specific version in your package.json file
+3. If you develop custom tooling around Tauri, you can import the CLI as a regular JavaScript module
+4. You can install the CLI using a JavaScript manager:
+
+## Recommended Browserlist
```

Comment:
We should update the Browserlist configuration for the vite guide with this one. `last 3 chrome versions` makes more sense than a fixed version anyway

Reply:
It is that valid for Vite? I wasn't able to test it

Reply:
Oh true it's not `browserlist` but some esbuild builtin, let me check 🤔

Reply:
Nope, just tried it and it errors out:
```
[vite:css-post] Invalid target "last 3 Chrome versions" in "--target=es2021,last 3 Chrome versions,safari13"
error during build:
Error: Invalid target "last 3 Chrome versions" in "--target=es2021,last 3 Chrome versions,safari13"
```